docs/UserGuide.adoc

```diff
@@ -505,21 +513,21 @@ __Constraints:__
 // tag::list[]
 ==== Listing tasks : `list` +
 
-To go back to the default list of tasks after a round of filtering, use:
+Want to go back to the default list of task after a search? Simply use the list command:
 
 Format: `list`
 // end::list[]
+
 // tag::select[]
 ==== Selecting a task: `select`
 
 To select the task identified by the index number used in the last task listing, use: +
 Format: `select INDEX`
 
-****
-* Selects the task at the specified `INDEX`.
-* The index refers to the index number shown in the most recent listing.
-* The index *must be a positive integer* `1, 2, 3, ...`
-****
+[NOTE]
+====
+The selected task will show involved people in the address book. If there are no involved personnel, there will be no changes in the displayed list of contacts.
```

Comment:
involved *contacts
Try making the first statement a little clearer. e.g. "All contacts involved in the selected task will be listed"